enterprise/frontend/src/metabase-enterprise/embedding/index.js

```diff
@@ -12,3 +14,10 @@ if (hasPremiumFeature("embedding")) {
   PLUGIN_ADMIN_SETTINGS.InteractiveEmbeddingSettings =
     InteractiveEmbeddingSettings;
 }
+
+/**
+ * We can't gate this component behind a feature flag, because SDK users could
+ * use the SDK without a valid license and doesn't contain any feature flags.
+ */
+PLUGIN_EMBEDDING.SimpleDataPicker = SimpleDataPicker;
```

Comment:
This is moved from `enterprise/frontend/src/metabase-enterprise/embedding-sdk/index.js`